.env.example

```
@@ -0,0 +1,71 @@
+# Reloaderoo Configuration (Development)
+MCPDEV_PROXY_LOG_LEVEL=debug
+MCPDEV_PROXY_RESTART_LIMIT=5
+MCPDEV_PROXY_AUTO_RESTART=true
+MCPDEV_PROXY_TIMEOUT=30000
+
+# Google OAuth Configuration
+FASTMCP_AUTH=fastmcp.server.auth.providers.google.GoogleProvider
+FASTMCP_AUTH_GOOGLE_CLIENT_ID=your-google-client-id.apps.googleusercontent.com
+FASTMCP_AUTH_GOOGLE_CLIENT_SECRET=your-google-client-secret
+FASTMCP_AUTH_GOOGLE_BASE_URL=https://your-server.com
+FASTMCP_AUTH_GOOGLE_REQUIRED_SCOPES="openid,https://www.googleapis.com/auth/userinfo.email"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)